nginx/ci/nginx.rake

```diff
@@ -0,0 +1,80 @@
+require 'ci/common'
+
+def nginx_version
+  ENV['FLAVOR_VERSION'] || '1.7.11'
+end
+
+def nginx_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/nginx_#{nginx_version}"
+end
+
+container_name = 'dd-test-nginx'
+container_port1 = 44441
+container_port2 = 44442
+
+namespace :ci do
+  namespace :nginx do |flavor|
+    task before_install: ['ci:common:before_install'] do
+      sh %(docker kill #{container_name} 2>&1 >/dev/null || true 2>&1 >/dev/null)
+      sh %(docker rm #{container_name} 2>&1 >/dev/null || true 2>&1 >/dev/null)
+    end
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements('nginx/requirements.txt',
+                           "--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      if nginx_version == "1.6.2"
+        repo = "centos/nginx-16-centos7"
```

Comment:
Any reason why we're not using the official [nginx](https://hub.docker.com/_/nginx/) image?